apps/test/unit/lib/kits/maker/boards/microBit/MicroBitBoardTest.js

```diff
@@ -27,200 +27,216 @@ function stubOpenSerialPort(board) {
   });
 }
 
-describe('MicroBitBoard', () => {
-  let board;
-
-  beforeEach(() => {
-    // Construct a board to test on
-    window.SerialPort = {};
-    board = new MicroBitBoard();
-    board.boardClient_ = new MicrobitStubBoard();
-    boardSetupAndStub(board);
-  });
-
-  afterEach(() => {
-    board = undefined;
-    sinon.restore();
-  });
-
-  describe('Maker Board Interface', () => {
-    itImplementsTheMakerBoardInterface(MicroBitBoard, board => {
+export function itMakesMicroBitComponentsAvailable(
```

Comment:
I was surprised to see test code like this being exported. Can you share some details about why it made more sense to export instead of pulling the code from FakeBoardTest into here?

Reply:
Sure! The set of tests from `itImplementsTheMakerBoardInterface` is imported from `MakerBoardTest.js` and then called on for both `FakeCPBoard` and `FakeMBBoard` in `FakeBoardTest.js`.  The function `itMakesCircuitPlaygroundComponentsAvailable` imported from`CircuitPlaygroundBoardTest.js` is also called on in `FakeBoardTest.js`. Thus, I was following this pattern and exported `itMakesMicroBitComponentsAvailable` so that we can check that the `FakeMBBoard` also passes this set of tests.

Reply:
Ok, there's something that's still feeling a little strange here, but I'll retain this as a non-blocking comment, especially since I can't put to words what's on my mind 😂

Reply:
I'm intrigued about why this feels a little strange to you and definitely open to discussion. I'll add a note to discuss during our next 1-1 mtg and am also open to refactoring these tests during the time set aside this summer for 'Maker Cleanup'.